python/paddle/fluid/tests/unittests/test_dist_base.py

```diff
@@ -566,12 +574,12 @@ def run_gpu_fleet_api_trainer(self, args):
         args.trainer_id = paddle.distributed.get_rank()
 
         # 3. init parallel env
-        if args.update_method == "nccl2":
+        if args.update_method == "nccl2" or "bkcl":
```

Comment:
this fleet is paddle.fluid.incubate.fleet, maybe need another test

Reply:
New paddle.distributed.fleet API interface is  imported again In this run_use_fleet_api_trainer function:
```
import paddle.distributed.fleet as fleet
import paddle.distributed.fleet.base.role_maker as role_maker
```
so , the unittest finally runs with new paddle.distributed.fleet API.